src/manager.js

```diff
@@ -84,13 +85,27 @@ AdManager.prototype.initGoogleTag = function() {
   this.googletag.pubads().addEventListener('slotOnload', adManager.onSlotOnload);
 
   this.initBaseTargeting();
+  if (this.options.amazon_enabled) {
+    this.initAmazonA9();
+  }
 
   this.initialized = true;
 
   this.loadAds();
 };
 
 /**
+ * initializes A9
+ *
+ * @returns undefined
+*/
+AdManager.prototype.initAmazonA9 = function() {
+  amznads.getAds('3076');
```

Comment:
Could make it part of base `bulbs` settings (https://github.com/theonion/omni/blob/86b2453ac1f5a269c006eb326844bbdb24e71bec/django-bulbs/bulbs/utils/middleware.py#L17).  Think it would be a good idea to make that account ID a config, and then put it in the `window.Bulbs.settings` object